crates/driver/src/domain/competition/order/mod.rs

```diff
@@ -232,7 +215,11 @@ pub enum BuyTokenBalance {
 /// [`Order`].
 #[derive(Debug)]
 pub struct Jit {
+    /// The maximum amount this order is allowed to sell when completely filled.
+    /// The actual executed amount depends on partial fills and the order side.
     pub sell: eth::Asset,
+    /// The minimum amount this order must buy when completely filled.
+    /// The actual executed amount depends on partial fills and the order side.
     pub buy: eth::Asset,
```

Comment:
JIT orders are always executed at exactly their limit price. So you can actually consider JIT orders amounts as flux.

Reply:
Oh, true. I overlooked that these comments could be slightly different for JIT orders as they are always liquidity orders.